gnomad_hail/resources/grch38/reference_data.py

```diff
@@ -33,6 +34,12 @@ def _import_purcell_5k(path) -> hl.Table:
     return p5k.key_by('locus')
 
 
+def _import_clinvar(**kwargs) -> hl.Table:
+    clinvar = import_sites_vcf(**kwargs)
+    clinvar = hl.vep(clinvar, vep_config_path(ref='GRCh38'))
```

Comment:
Should we use `gnomad_hail.utils.generic.vep_or_lookup_vep` for this?

Reply:
Good point! I forgot we had it for 38.